test/TestUtilities/Test.Utility/Signing/SignedArchiveTestUtility.cs

```diff
@@ -134,13 +120,16 @@ public static class SignedArchiveTestUtility
         /// Sign a package for test purposes.
         /// This does not timestamp a signature and can be used outside corp network.
         /// </summary>
-        private static async Task SignPackageAsync(TestLogger testLogger, X509Certificate2 certificate, SignedPackageArchive signPackage)
+        private static async Task SignPackageAsync(TestLogger testLogger, X509Certificate2 certificate, string inputPackagePath, string outputPackagePath)
         {
             var testSignatureProvider = new X509SignatureProvider(timestampProvider: null);
-            var signer = new Signer(signPackage, testSignatureProvider);
             var request = new AuthorSignPackageRequest(certificate, HashAlgorithmName.SHA256);
+            using (var signerRequest = new SignerOptions(inputPackagePath, outputPackagePath, false, testSignatureProvider, request, testLogger))
```

Comment:
`signerOptions`

Add a named parameter, inline comment, or constant for `false`.